config/default

```diff
@@ -21,11 +17,11 @@ export ign_api_key = 9qxhimhvm24i40p8j86n7de3
 export bing_api_key = AudizIfCd3NAdt91ubJMGkMI-swfHxe1R-_U7KiLxCHqepDy70txQ-_-89_eevxc
 
 export ganalytics_key = FIXME
+export recaptcha_site_key = 6LdWkR4TAAAAAJEW4VaASKHU55wP42yYNy7Mbgdb
```

Comment:
Is it normal that this key is not the same than for the API: https://github.com/c2corg/v6_api/blob/2ce9c6425e6bd85a7d294a9e4776be5e19ad4f5e/config/default#L75